### ISSUE TITLE
Unbound 

### ISSUE THREAD

**Issue by louiemondot** [heroku/buildpacks-jvm#211]
I'm using Gradle 7.3, JDK 17.0.1, Spring Boot 2.5.6 and building using: `./gradlew bootBuildImage --builder heroku/spring-boot-buildpacks --imageName=...`.   The script fails when installing Java:

```
> Task :bootBuildImage
Building image 'registry.heroku.com/.../web:latest'

 > Pulling builder image 'docker.io/heroku/spring-boot-buildpacks:latest' ..................................................
 > Pulled builder image 'heroku/spring-boot-buildpacks@sha256:730e3dc2ee4a0ecd05813510111f1e6deffc60a65bb602e393dc5f4b9d1d13b8'
 > Pulling run image 'docker.io/heroku/pack:18' ..................................................
 > Pulled run image 'heroku/pack@sha256:774c481940ede4ce59caa81401b52ab665f1a55766f704268db2308958198557'
 > Executing lifecycle version v0.10.1
 > Using build cache volume 'pack-cache-ee04f47d3cc8.build'

 > Running creator
    [creator]     ===> DETECTING
    [creator]     Warning: Warning: buildpack heroku/jvm has a "version" key. This key is deprecated in build plan requirements in buildpack API 0.3. "metadata.version" should be used instead
    [creator]     heroku/jvm         0.1.0
    [creator]     heroku/spring-boot 0.2.2
    [creator]     ===> ANALYZING
    [creator]     ===> RESTORING
    [creator]     ===> BUILDING
    [creator]
    [creator]     [Installing Java]
    [creator]     /cnb/buildpacks/heroku_jvm/0.1.0/lib/jvm.sh: line 104: jdkUrl: unbound variable
    [creator]     ERROR: failed to build: exit status 1

> Task :bootBuildImage FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':bootBuildImage'.
> Builder lifecycle 'creator' failed with status code 145
```

I do have some yarn tasks that build prior and they build fine.

Not sure why this is failing.  Any ideas?

**Comment by maechler** [comment]
We got the same error with a completely fresh Spring Boot application from https://start.spring.io/. We are using Gradle `7.3.2`, JDK `17.0.1` and Spring Boot `2.6.2`.

**Comment by edmorley** [comment]
@Malax After #272 (which is pending release), `buildpacks/jvm/lib/jvm.sh` no longer exists. Can this be closed out now?

**Comment by Malax** [comment]
Yeah, I'll close this for now since the buildpack dramatically changed. Feel free to open a new issue should this issue still be present in the current version (`1.0.0`).
